File: leetcode2026/2026_01/leetcode2026-01-25.cpp
#include <iostream>
#include <vector>
#include <climits>
#include <algorithm>
using namespace std;

class Solution {
public:
    int minimumDifference(vector<int>& nums, int k) {
        sort(nums.begin(), nums.end());
        int n = nums.size();
        int res = INT_MAX;
        for (int i=0; i<=n-k; i++){
            res = min(res, nums[i+k-1]-nums[i]);
        }
        return res;
    }
};

int main(){
    Solution s;
    vector<int> nums = {90};
    int k = 1;
    cout << s.minimumDifference(nums, k) << endl;
}

File: leetcode2026/2026_01/leetcode2026-01-25.py
from typing import List

class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        nums.sort()
        n = len(nums)
        res = float("inf")
        for i in range(n-k+1):
            cur = nums[i+k-1]-nums[i]
            if cur < res:
                res = cur
        return res

if __name__ == "__main__":
    s = Solution()
    nums = [90]
    k = 1
    print(s.minimumDifference(nums, k))
